docs/track.rst

```diff
@@ -402,6 +402,7 @@ With the operation type ``search`` you can execute `request body searches <http:
         2. Rally will not attempt to serialize the parameters and pass them as is. Always use "true" / "false" strings for boolean parameters (see example below).
 
 * ``body`` (mandatory): The query body.
+* ``detailed-results`` (optional, defaults to ``true``): Records more detailed meta-data about queries. As it analyzes the corresponding response in more detail, this might incur additional overhead which can skew measurement results. Set this value to ``false`` for queries that return within single digit milliseconds to increase measurement accuracy. This flag is ineffective for scroll queries.
```

Comment:
Perhaps we should mention some important fields like `took` and `hits` fields from Elasticsearch that won't be captured when ``detailed-results:false`; might save some head scratching for people who disable this and wonder what is it that they'll be losing?

I see it is included in the [migration docs](https://github.com/elastic/rally/pull/935/commits/4c2e78a125a6b98fff25010c16066275e61f53d2)

Reply:
I pushed https://github.com/elastic/rally/pull/935/commits/5ab95230d29f9a8c256551c7e60e900be8ead364.